src/record/recordExpressionEvaluator/recordExpressionEvaluator.test.ts

```diff
@@ -136,7 +136,7 @@ describe('RecordExpressionEvaluator', () => {
       result: 'Extra prop1 item 2-2',
       node: 'cluster.plot[1].plot_id',
     },
-    // categoryItemProp: unexisting prop or code
+    // categoryItemProp: non existing prop or code
```

Comment:
this one really makes the difference!

Reply:
thanks for this useful comment. the IDE was marking as a wrong word, and I corrected it.